plenum/__init__.py

```diff
@@ -9,6 +9,7 @@
     raise ImportError("Python 3.5.0 or later required.")
 
 import os   # noqa
```

Comment:
A tech debt from previous PR:
please rename `test_no_ledgerId_fails`

Reply:
Which branch are you looking at? The test had been renamed long ago to `test_no_ledgerId_passes`

Reply:
master
https://github.com/hyperledger/indy-plenum/blob/master/plenum/test/input_validation/test_client_get_txn_op.py#L14

Reply:
But why are looking at master? The changes are to be made in `config-ledger-plenum`